.github/workflows/CI-BUILD.yml

```
@@ -162,40 +162,53 @@
         with:
           persist-credentials: false
-      - name: Set up Python
+      - id: buildpy
+        name: Set up Python
         uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
         with:
           python-version: ${{ matrix.python-version }}
+      - id: output_python
+        name: "bootstrap Python"
+        shell: bash
+        run: |
```

actionlint (1.7.7)

property "cp313" is not defined in object type {buildpy: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)